purchase_requisition_full_bid_order_generator/README.rst

```diff
@@ -0,0 +1,16 @@
+Purchase Requisition Full Bid Order Generator
+=============================================
+
+This module allows you to request a quotation for each suppliers that have all the products/services included in the bid.
```

Comment:
Incluye al final: `For that, the supplier must be included in the "Suppliers" section of the "Procurement" tab of the products.`